elephant/asset/asset.py

```diff
@@ -1977,11 +1983,15 @@ class ASSET(object):
 
           fully disjoint.
 
+    See Also
+    --------
+    conv.BinnedSpikeTrain
```

Comment:
```suggestion
    :class:`elephant.conversion.BinnedSpikeTrain`
```

Get working link to `BinnedSpikeTrain` on readthedocs.

Reply:
Fixed in commit [8a090e8](https://github.com/NeuralEnsemble/elephant/pull/585/commits/8a090e88466b547206ae495e10c6e6f51dbe0dfe)